index.html

```diff
@@ -3,7 +3,6 @@
 
 <head>
   <meta charset="UTF-8">
-  <link rel="icon" href="/favicon.ico">
```

Comment:
Why did you remove favicon? You don't want any?

Reply:
ok, I pasted it back to the project